Description: Debianize chronyd-restricted.service
Author: Vincent Blut <vincent.debian@free.fr>
Forwarded: not-needed
Last-Update: 2023-08-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: chrony/examples/chronyd-restricted.service
===================================================================
--- chrony.orig/examples/chronyd-restricted.service
+++ chrony/examples/chronyd-restricted.service
@@ -14,10 +14,10 @@ ConditionCapability=CAP_SYS_TIME
 [Service]
 Type=forking
 PIDFile=/run/chrony/chronyd.pid
-EnvironmentFile=-/etc/sysconfig/chronyd
-ExecStart=/usr/sbin/chronyd -U $OPTIONS
+EnvironmentFile=-/etc/default/chrony
+ExecStart=/usr/sbin/chronyd -U $DAEMON_OPTS
 
-User=chrony
+User=_chrony
 LogsDirectory=chrony
 LogsDirectoryMode=0750
 RuntimeDirectory=chrony
